src/components/breadcrumb/breadcrumb.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from '../links/links';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+import { BreadcrumbText } from '../typography/typography';
+import { theme } from '../layout/theme';
+
+const BreadcrumbContainer = styled.ol`
+  margin: 16px 24px 0;
+  list-style: none;
+  display: flex;
+  flex-wrap: nowrap;
+  overflow: hidden;
+  padding: 0;
+`;
+
+const BreadcrumbListItem = styled(BreadcrumbText)`
+  white-space: nowrap;
+  color: ${theme.palette.text.breadcrumb};
+  &:not(:last-child) {
+    margin-right: 7px;
+    ::after {
+      content: '›';
+      margin-left: 7px;
+      display: inline-flex;
+      flex: 0 0 auto;
+    }
+  }
+  :last-child {
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
```

Comment:
This is a very nice touch. Thanks for thinking about this 🙏 
![image](https://user-images.githubusercontent.com/7970458/141332150-a8e63bd8-6db0-4d9c-9a66-ac7804276d60.png)